src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -490,7 +490,14 @@ public void TestUserCredentialsPropertiesOnWindows()
                 p.StartInfo.UserName = username;
                 p.StartInfo.PasswordInClearText = password;
 
-                hasStarted = p.Start();
+                try
+                {
+                    hasStarted = p.Start();
+                }
+                catch (Win32Exception ex) when (ex.NativeErrorCode == ERROR_SHARING_VIOLATION)
```

Comment:
I guess this covers "The process cannot access the file because it is being used by another process.", Does it cover "Access is denied." as well?

Reply:
@Jozkee good catch! I've extended the current logic that was granting access to the executable file to also provide access to the working directory. I hope it fixes the issue. PTAL